.travis.yml

```diff
@@ -0,0 +1,28 @@
+Check on http://lint.travis-ci.org/ after modifying it!
+sudo: false
+language: python
+env:
+  - PYVERSION=3.5 USE_CYTHON=TRUE
+before_install:
+  - "export DISPLAY=:99.0"
```

Comment:
IIRC, it's for matplotlib, I remember seeing it somewhere in the
#scikit-bio ML.

On (Jun-30-16| 6:08), Antonio Gonzalez wrote:
> > @@ -0,0 +1,28 @@
> > +Check on http://lint.travis-ci.org/ after modifying it!
> > +sudo: false
> > +language: python
> > +env:
> > +  - PYVERSION=3.5 USE_CYTHON=TRUE
> > +before_install:
> > +  - "export DISPLAY=:99.0"
> 
> I guess it is for ETE ... 
> 
> ---
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/gneiss/pull/2/files/d9d026e68a7b2594ef2b01bb9689afe9b2f0e1dc#r69128111